conftest.py

```diff
@@ -0,0 +1,37 @@
+from Matrix import Matrix
+import pytest
+
+@pytest.fixture
+def add_result():
+    f_m = Matrix([[0, 1],[0, 2]])
+    s_m = Matrix([[0, 1],[0, 2]])
+    result_m = Matrix([[0, 2],[0, 4]])
+    return [f_m, s_m, result_m]
+
+
+@pytest.fixture
+def sub_result():
+    f_m = Matrix([[0, 1],[0, 2]])
+    s_m = Matrix([[0, 1],[0, 2]])
```

Comment:
You use those two matrices in 4 tests. Every developer **must** follow **_[DRY](https://habr.com/ru/companies/itelma/articles/546372/)_** principle

create two new fixtures like this:

```python
@pytest.fixture
def matrix_one():
    first_matrix = Matrix([[0, 1],[0, 2]])
    return first_matrix
@pytest.fixture
def matrix_two():
    first_matrix = Matrix([[0, 1],[0, 2]])
    return first_matrix
```
**and use them in other fixtures**